roguepygame/root.py

```diff
@@ -1,11 +1,13 @@
-from typing import Optional, Type, Any, Callable, TYPE_CHECKING, Protocol
+import collections.abc
+from typing import Optional, Type, Any, Callable, TYPE_CHECKING, Protocol, Iterator, TypeVar
 
 import pygame
 import constants as const
 if TYPE_CHECKING:
     import game
     class SupportsEvents(Protocol):
         def events(self, event: pygame.event.Event) -> None: ...
+game_object_type = TypeVar('game_object_type', bound='GameObject')
```

Comment:
I've seen the convention to name these with primary underscore (sometimes) and treat as constant variables with  UPPER_SNAKE_CASE (always?).
```suggestion
_GAME_OBJECT_TYPE = TypeVar('game_object_type', bound='GameObject')
```
If renaming, don't forget to rename its occurrences.